lib/pure/typetraits.nim

```diff
@@ -107,6 +107,28 @@ since (1, 3, 5):
 
 import std/macros
 
+macro len*(T: typedesc[enum]): int =
+  ## Returns the number of items in the enumeration `T`.
```

Comment:
nit: s/enumeration/enum/ since that's the terminology used

Reply:
Fixed.